COPYING

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
@cosimoc what's the take on this ? do we want our code to be reuseable in non-free software ?
as i'm mostly the author i'd rather it be gplv3